test-cases/tests/documentation/i16-verify-partial-ag-rhmi-config.md

```diff
@@ -0,0 +1,28 @@
+---
+environments:
+  - osd-fresh-install
+tags:
+  - manual-selection
+---
```

Comment:
```suggestion
  - manual-selection
targets:
  - 2.7.0
---
```
Just need to add also the 2.7.0 target otherwise it will not be executed during the 2.7.0 release